### PR TITLE
Webhook caches

### PR DESCRIPTION
**Pull request by jwilander**
mattermost/enterprise#89

### CODE REVIEW

**Comment by coreyhulen** [comment]
Unit test on the cluster purge by channel name needed.

File: api/post.go [hunk 1]
```diff
@@ -300,6 +297,9 @@ func handlePostEvents(c *Context, post *model.Post, triggerWebhooks bool) {
 		channel = result.Data.(*model.Channel)
 	}
 
+	InvalidateCacheForChannel(channel)
```
Comment (coreyhulen) [inline api/post.go]: is there a reason why this is moved down?

I would guess the more time between this and publish would be better.

Comment (jwilander) [inline api/post.go]: So I could pass in the channel rather than just the channel id

Comment (coreyhulen) [inline api/post.go]: ahh..ok.

**Pull request merged**